build-scripts/build-parameters.cake

```diff
@@ -0,0 +1,312 @@
+    //	? $"\"{ZipPackage},{NuGetPackage},{ChocolateyPackage},{MetadataPackage}\""
+    //       : $"\"{ZipPackage},{NuGetPackage}\"";
+
+    //public string MyGetPushUrl => MYGET_PUSH_URL;
+    //public string NuGetPushUrl => NUGET_PUSH_URL;
+    //public string ChocolateyPushUrl => CHOCO_PUSH_URL;
+
+    //public string MyGetApiKey { get; }
+    //public string NuGetApiKey { get; }
+    //public string ChocolateyApiKey { get; }
+
+    //   public string BranchName => BuildVersion.BranchName;
+    //public bool IsReleaseBranch => BuildVersion.IsReleaseBranch;
+
+    //public bool IsPreRelease => BuildVersion.IsPreRelease;
+    //public bool ShouldPublishToMyGet =>
+    //	!IsPreRelease || LABELS_WE_PUBLISH_ON_MYGET.Contains(BuildVersion.PreReleaseLabel);
+    //public bool ShouldPublishToNuGet =>
+    //	!IsPreRelease || LABELS_WE_PUBLISH_ON_NUGET.Contains(BuildVersion.PreReleaseLabel);
+    //public bool ShouldPublishToChocolatey =>
+    //	!IsPreRelease || LABELS_WE_PUBLISH_ON_CHOCOLATEY.Contains(BuildVersion.PreReleaseLabel);
+    //public bool IsProductionRelease => ShouldPublishToNuGet || ShouldPublishToChocolatey;
+
+    //public bool UsingXBuild { get; }
+    //public MSBuildSettings MSBuildSettings { get; }
+    //public XBuildSettings XBuildSettings { get; }
+    //public NuGetRestoreSettings RestoreSettings { get; }
+
+    //public string[] SupportedEngineRuntimes => new string[] {"net40", "netcoreapp2.1"};
+    //public string[] SupportedCoreRuntimes => IsRunningOnWindows
+    //	? new string[] {"net40", "net35", "netcoreapp2.1", "netcoreapp1.1"}
+    //	: new string[] {"net40", "net35", "netcoreapp2.1"};
+    //public string[] SupportedAgentRuntimes => new string[] { "net20", "net40", "netcoreapp2.1", "netcoreapp3.1", "net5.0" };
+
+    //public string ProjectUri => "https://github.com/TestCentric/testcentric-gui";
+    //public string WebDeployBranch => "gh-pages";
+    //public string GitHubUserId => "charliepoole";
+    //public string GitHubUserEmail => "charliepoole@gmail.com";
+    //public string GitHubPassword { get; }
+    //public string GitHubAccessToken { get; }
+
+    private void Validate()
+	{
+		var validationErrors = new List<string>();
+
+		//if (TasksToExecute.Contains("PublishPackages"))
+		//{
+		//	if (ShouldPublishToMyGet && string.IsNullOrEmpty(MyGetApiKey))
+		//		validationErrors.Add("MyGet ApiKey was not set.");
+		//	if (ShouldPublishToNuGet && string.IsNullOrEmpty(NuGetApiKey))
+		//		validationErrors.Add("NuGet ApiKey was not set.");
+		//	if (ShouldPublishToChocolatey && string.IsNullOrEmpty(ChocolateyApiKey))
+		//		validationErrors.Add("Chocolatey ApiKey was not set.");
+		//}
+
+		//if (TasksToExecute.Contains("CreateDraftRelease") && (IsReleaseBranch || IsProductionRelease))
+		//{
+		//	if (string.IsNullOrEmpty(GitHubAccessToken))
+		//		validationErrors.Add("GitHub Access Token was not set.");		
+		//}
+
+		if (validationErrors.Count > 0)
+		{
+			DumpSettings();
```

Comment:
Currently, this is dead code, as we don't add anything to `validationErrors`

Reply:
I'll remove.